tests/test_association_task.py

```diff
@@ -23,529 +23,86 @@
 import pandas as pd
 import unittest
 
-from lsst.afw.cameraGeom.testUtils import DetectorWrapper
-import lsst.afw.geom as afwGeom
-import lsst.afw.image as afwImage
-import lsst.afw.table as afwTable
-import lsst.daf.base as dafBase
 import lsst.geom as geom
-import lsst.sphgeom as sphgeom
 import lsst.utils.tests
 
-from lsst.ap.association import \
-    AssociationTask, \
-    make_dia_source_schema, \
-    make_dia_object_schema
-
-
-def create_test_points(point_locs_deg,
-                       wcs=None,
-                       start_id=0,
-                       schema=None,
-                       scatter_arcsec=1.0,
-                       indexer_ids=None,
-                       associated_ids=None):
-    """Create dummy DIASources or DIAObjects for use in our tests.
-    Parameters
-    ----------
-    point_locs_deg : array-like (N, 2) of `float`s
-        Positions of the test points to create in RA, DEC.
-    wcs : `lsst.afw.geom.SkyWcs`
-        Wcs to convert RA/Dec to x/y if provided.
-    start_id : `int`
-        Unique id of the first object to create. The remaining sources are
-        incremented by one from the first id.
-    schema : `lsst.afw.table.Schema`
-        Schema of the objects to create. Defaults to the DIASource schema.
-    scatter_arcsec : `float`
-        Scatter to add to the position of each DIASource.
-    indexer_ids : `list` of `ints`s
-        Id numbers of pixelization indexer to store. Must be the same length
-        as the first dimension of point_locs_deg.
-    associated_ids : `list` of `ints`s
-        Id numbers of associated DIAObjects to store. Must be the same length
-        as the first dimension of point_locs_deg.
-    Returns
-    -------
-    test_points : `lsst.afw.table.SourceCatalog`
-        Catalog of points to test.
-    """
-    if schema is None:
-        schema = make_dia_source_schema()
-    sources = afwTable.SourceCatalog(schema)
-
-    for src_idx, (ra, dec,) in enumerate(point_locs_deg):
-        src = sources.addNew()
-        src['id'] = src_idx + start_id
-        coord = geom.SpherePoint(ra, dec, geom.degrees)
-        if scatter_arcsec > 0.0:
-            coord = coord.offset(
-                np.random.rand() * 360 * geom.degrees,
-                np.random.rand() * scatter_arcsec * geom.arcseconds)
-        if indexer_ids is not None:
-            src['pixelId'] = indexer_ids[src_idx]
-        if associated_ids is not None:
-            src['diaObjectId'] = associated_ids[src_idx]
-        src.setCoord(coord)
-
-        if wcs is not None:
-            xyCentroid = wcs.skykToPixel(coord)
-            src.set("x", xyCentroid.getX())
-            src.set("y", xyCentroid.getY())
-
-    return sources
-
-
-def create_test_points_pandas(point_locs_deg,
```

Comment:
Why get rid of this function? Even with the tests being completely rewritten, this looks like it would have been useful.

Reply:
It was easy enough to create the sources in setup using pandas. The functions here are leftovers from when association handled interactions with the Apdb and used afw tables instead.

Reply:
Yes, but you end up creating several different sets of points, and having to copy the code almost verbatim.

Reply:
I mean sure, but it's not like that code is complicated at all. I'm inclined to leave it as it is.